src/test/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesControllerTest.java

```diff
@@ -52,6 +53,7 @@ public void createMissingImageDeliverySuccessfully() {
 		assertThat(response.getStatusCode(), is(HttpStatus.CREATED));
 		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getCompanyName(), response.getBody().getCompanyName());
 		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getCompanyNumber(), response.getBody().getCompanyNumber());
+		assertEquals(MISSING_IMAGE_DELIVERIES_DTO.getFilingHistoryType(), response.getBody().getFilingHistoryType());
```

Comment:
This shouldn't have tabs in it, it should use spaces instead. If the entire class is like this, we should raise the formatting as a separate Jira issue. If not, this should be re-formatted.